docs/recipes/SimplifiedGetCurrentBalanceFromAccount.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 1
+title: Simplified Get balance of the connected account
+description: Learn how to retrieve and display the ETH balance of the connected account in your dApp.
+---
+
+# Get the Current Balance of the Connected Account
+
+This recipe shows how to fetch and display the ETH balance of the currently connected account in your decentralized application. After the recipe code, you'll find an implementation guide that walks you through the steps of creating this recipe yourself.
+
+```tsx
+import type { NextPage } from "next";
+import { useAccount } from "wagmi";
+import { Address, Balance } from "~~/components/scaffold-eth";
+
+const ConnectedAddressBalance: NextPage = () => {
```

Comment:
Check the components we have in SE-2. We export directly from here:

`export const ConnectedAddressBalance = ()... `

Also remove "NextPage".

Reply:
Fixed now, ty!